test/SILGen/property_wrapper_coroutine.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %target-swift-emit-silgen %s | %FileCheck %s
+
+@propertyWrapper
+struct TestWrapper<ValueType> {
+  var wrappedValue: ValueType
+}
+
+struct State {
+  @TestWrapper var values: [String] = []
+}
+
+var state = State()
+state.values = Array(repeating: "", count: 20000)
+
+for i in 0..<20000 {
+  // CHECK: bb3([[INDEX:%.*]] : $Int):
+  // CHECK:  [[BEGIN_ACCESS_MODIFY:%.*]] = begin_access [modify] [dynamic] {{%.*}} : $*State
+  // CHECK:  [[REF_MODIFY:%.*]] = function_ref @$s26property_wrapper_coroutine5StateV6valuesSaySSGvM : $@yield_once @convention(method) (@inout State) -> @yields @inout Array<String>
+  // CHECK:  ([[RES1:%.*]], {{%.*}}) = begin_apply [[REF_MODIFY]]([[BEGIN_ACCESS_MODIFY]]) : $@yield_once @convention(method) (@inout State) -> @yields @inout Array<String>
+  // CHECK:  [[REF_ARRAY_SUBSCRIPT:%.*]] = function_ref @$sSayxSiciM : $@yield_once @convention(method) <τ_0_0> (Int, @inout Array<τ_0_0>) -> @yields @inout τ_0_0
+  // CHECK:  ({{%.*}}, {{%.*}}) = begin_apply [[REF_ARRAY_SUBSCRIPT]]<String>([[INDEX]], [[RES1]]) : $@yield_once @convention(method) <τ_0_0> (Int, @inout Array<τ_0_0>) -> @yields @inout τ_0_0
+  state.values[i] = String(i)
+}
```

Comment:
Please just put something like `state.values[1000] = "hello"` in a function by itself that can be tested without potentially-unstable dependencies on the basic-block numbering of the global scope.